python-code/basics-of-python/four.py

```diff
@@ -0,0 +1,27 @@
+import re
+
+class ValidatePassword:
+    def __init__(self, password_arr):
+        self.password_arr = password_arr
+        
+    def validate(self, password):
+        pattern = r"^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[$#@]).{6,12}$"
```

Comment:
This can be made more readable by storing the different patterns in appropriately named variables